varnish/tests/test_varnish.py

```diff
@@ -29,6 +29,15 @@ def test_check(aggregator, instance, dd_run_check):
     aggregator.assert_metrics_using_metadata(metadata_metrics, check_submission_type=True)
 
 
+def test_check_invalid_varnishadm(aggregator, instance, dd_run_check):
+    # Note: this test may prompt you to enter a password when locally, if so, enter anything and it will pass
```

Comment:
Why does it ask for a password locally?

Reply:
The code uses `sudo`: https://github.com/DataDog/integrations-core/blob/master/varnish/datadog_checks/varnish/varnish.py#L157